app/indexers/hyrax/admin_set_indexer.rb

```diff
@@ -8,6 +8,12 @@ def generate_solr_document
       super.tap do |solr_doc|
         # Makes Admin Sets show under the "Admin Sets" tab
         solr_doc['generic_type_sim'] = ['Admin Set']
+
+        solr_doc['alt_title_sim']     = object.alt_title
```

Comment:
Are the changes in the suffix here deliberate?  From what I can tell `stored_searchable` should map to `_tesim` which `alt_title` and `description` were both previously indexed as.

Reply:
hmmmm... no, i don't think changes are intended. When working on this I generated a `SolrDocument` using the old approach, and re-implemented (i think?) identically.

Maybe we need an integration test?

Reply:
I finally got around to shimming in a model test for the indexing behavior.

good catch on the keys, not sure how i got that wrong initially.